src/nvim/charset.c

```diff
@@ -84,13 +84,13 @@ int buf_init_chartab(buf_T *buf, int global)
   int c2;
```

Comment:
You changed the parameter `global` to boolean on line 72. Also change this definition to match.